### PR TITLE
Pizza Tweaks

### PR DESCRIPTION
**Pull request by crunchypretzels**
Just a little something to learn recipe code.

Ports most of the pizza sprites from /tg/- thanks, guys- and their respective boxes. I've changed the boxes to match Aurora better- they have a little Orion logo on them.

Also adds a recipe for pepperoni (1u salt, 1u space spice, 1 sausage in a skillet) and pepperoni pizza (2 tomatoes, 1 flat dough, 1 pepperoni stick, and 1 cheese wedge) in the oven.

Also adds a couple unused pizza boxes for a future project. 

All credit to the talented spriters on /tg/ for the pizza and pizza boxes. I only made the pizza crunch and pepperoni stick.

image

### CODE REVIEW

**Comment by crunchypretzels** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -3279,6 +3288,28 @@
...
+	desc = "Who doesn't love a good pepperoni pizza after a hard day in the Chainlink?"
+	icon_state = "pepperonipizza"
+	slice_path = /obj/item/reagent_containers/food/snacks/pepperonipizzaslice
+	slices_num = 6
+	center_of_mass = list("x"=16, "y"=11)
+	reagents_to_add = list(/singleton/reagent/nutriment = 10, /singleton/reagent/nutriment/protein = 44, /singleton/reagent/nutriment/protein/cheese = 10, /singleton/reagent/drink/tomatojuice = 6)
+	reagent_data = list(/singleton/reagent/nutriment = list("pizza crust" = 10, "tomato" = 10, "pepperoni" = 15))
+	bitesize = 2
+
+/obj/item/reagent_containers/food/snacks/pepperonipizzaslice
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/food/snacks.dm:3302]: Nope. Unacceptable item.  Blocked. Maintainer discussion. DNM. Feedback thread required

**Review by NonQueueingMatt — APPROVED** [review]

File: code/game/objects/random/random.dm [hunk 2]
```diff
@@ -1988,7 +1988,8 @@
 		/obj/item/pizzabox/vegetable,
 		/obj/item/pizzabox/mushroom,
 		/obj/item/pizzabox/meat,
-		/obj/item/pizzabox/pineapple
+		/obj/item/pizzabox/pineapple,
```
Comment (NonQueueingMatt) [inline code/game/objects/random/random.dm:1991]: GET IT OUT

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 1 above)
Comment (crunchypretzels) [reply code/modules/reagents/reagent_containers/food/snacks.dm:3302]: The greatest fast food chain ever to exist with history spanning centuries and tens of thousands of locations open worldwide. The peeperoni pizza is a delight for the senses.

File: code/game/objects/random/random.dm (hunk 2 above)
Comment (crunchypretzels) [reply code/game/objects/random/random.dm:1991]: could i just get a little macaroni and gravy?

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 3]
```diff
@@ -3279,6 +3288,28 @@
...
+	name = "pepperoni pizza slice"
+	desc = "A slice of pepperoni pizza."
+	icon_state = "pepperonipizzaslice"
+	filling_color = "#BAA14C"
+	bitesize = 2
+	center_of_mass = list("x"=18, "y"=13)
+
+/obj/item/reagent_containers/food/snacks/pepperonipizzaslice/filled
+	reagents_to_add = list(/singleton/reagent/nutriment = 2, /singleton/reagent/nutriment/protein = 7, /singleton/reagent/nutriment/protein/cheese = 2, /singleton/reagent/drink/tomatojuice = 2)
+	reagent_data = list(/singleton/reagent/nutriment = list("pizza crust" = 5, "tomato" = 5))
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/food/snacks.dm:3312]:
```suggestion
	reagent_data = list(/singleton/reagent/nutriment = list("pizza crust" = 5, "tomato" = 5))

```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**